cli/shared/src/main/scala/com/lightbend/rp/reactivecli/docker/package.scala

```diff
@@ -21,4 +21,15 @@ package object docker {
   val DockerDefaultRegistry = "registry.hub.docker.com"
   val DockerDefaultLibrary = "library"
   val DockerDefaultTag = "latest"
+
+  // Sometimes authentication credentials store different address than our default server,
```

Comment:
Is it possible our `DockerDefaultRegistry` is wrong? Originally I just kinda hacked at that until I found the value that seemed to be right.